vmngclient/session.py

```diff
@@ -344,11 +345,15 @@ def get_virtual_session_id(self, tenant_id: str) -> str:
         response = self.post(url_path)
         return response.json()["VSessionId"]
 
-    def logout(self) -> vManageResponse:
-        if self.api_version >= Version("20.12"):
-            return self.post("/logout")
+    def logout(self) -> Optional[vManageResponse]:
+        if version := self.api_version:
+            if version >= Version("20.12"):
+                return self.post("/logout")
+            else:
+                return self.get("/logout")
```

Comment:
Can we one-liner here ? I also suggest return None if version != self.api_version and then follow happy flow